databox/core/lib/ServerHooks.ts

```diff
@@ -0,0 +1,132 @@
+import { IDataboxApis } from '@ulixee/databox-interfaces/IDataboxApis';
+import IDataboxPackage from '@ulixee/databox-interfaces/IDataboxPackage';
+import PackageRegistry from './PackageRegistry';
+import IDataboxModuleRunner from '../interfaces/IDataboxModuleRunner';
+import ConnectionToClient from '@ulixee/net/lib/ConnectionToClient';
+import ITransportToClient from '@ulixee/net/interfaces/ITransportToClient';
+import IDataboxManifest from '@ulixee/databox-interfaces/IDataboxManifest';
+import env from '../env';
+import LocalDataboxProcess from './LocalDataboxProcess';
+
+type IDataboxConnectionToClient = ConnectionToClient<IDataboxApis, {}>;
+
+export default class ServerHooks {
+  public static connections = new Set<IDataboxConnectionToClient>();
```

Comment:
Should this be static? Seems like we've moved the codebase to not have static classes except when unavoidable. It seems like Core/index might be the static here

Reply:
I don't think I changed this; it was just a file rename.

Reply:
I think you ended up moving this, so doesn't matter. But you created a new file, so in this commit, we had a static core/index and a static core/lib/ServerHooks. That was all I meant. Added a second, seemingly unnecessary static.